packages/rfw/lib/src/flutter/material_widgets.dart

```diff
@@ -50,6 +51,22 @@ import 'runtime.dart';
 ///  * [VisualDensity] is represented in the manner described in the documentation
 ///    of the [ArgumentDecoders.visualDensity] method.
 ///
+/// Some features have changed in the underlying Material library and are
+/// therefore no longer supported, including:
+///
+///  * The ButtonBar widget in package:flutter/material.dart is planned to be
+///    deprecated in favor of the OverflowBar widget. The ButtonBar widget in
+///    rfw package uses the OverflowBar widget internally for backward compatibility.
+///    The ButtonBar widget in rfw package is not deprecated and will continue to
+///    be supported. As a result, the following ButtonBar parameters are no longer
+///    supported:
+///
+///    * buttonMinWidth
+///    * buttonHeight
+///    * buttonAlignedDropdown
+///
+///    It is recommended to use the OverflowBar widget.
+///
```

Comment:
this all needs to be markdown with hyperlinks and so on (sorry if i wasn't clear in the earlier comment).

What I mean is e.g. `ButtonBar` needs to be `[ButtonBar]`, `package:flutter/material.dart` needs to be whatever the magic incantation is to reference the `material` library (maybe just `[material]`?), `rfw` needs to either be in backticks or uppercase, the parameters in the list should be in backticks, etc.

Reply:
Ahh my bad. Thanks for the suggestion!

`package:flutter/material.dart` can be simply referenced as "Flutter's material library". We reference like this in the framework too in various class.